data/water/wetlands/index.html

```diff
@@ -11,33 +11,21 @@
+    - url: https://drive.google.com/file/d/0ByStJjVZ7c7mbnlUUTNWbWdOclU
       label: 'Wetlands: File Geodatabase'
-    - url: https://drive.google.com/a/utah.gov/uc?id=0ByStJjVZ7c7mV084TkJ2S0JmOXc&export=download
+    - url: https://drive.google.com/drive/u/0/folders/0ByStJjVZ7c7mNW9iXzJrQUYyV0U
       label: 'Wetlands: Shapefile'
-    - url: https://drive.google.com/a/utah.gov/uc?id=0ByStJjVZ7c7maWN0RUVCWXVOdTA&export=download
-      label: 'Wetlands Mapping Projects: File Geodatabase'
-    - url: https://drive.google.com/a/utah.gov/uc?id=0ByStJjVZ7c7maUhMQWlVZzg1dm8&export=download
-      label: 'Wetlands Mapping Projects: Shapefile'
   updates:
-    - This feature class was first published in March 2014 and will be updated yearly
-WetlandsNWI:
-  downloads:
-    - url: https://drive.google.com/a/utah.gov/uc?id=0ByStJjVZ7c7mTGNRTTZZMkZkUDA&export=download
-      label: 'Wetlands NWI: File Geodatabase'
-    - url: https://drive.google.com/a/utah.gov/uc?id=0ByStJjVZ7c7mV0E1cGs0NWxodGM&export=download
-      label: 'Wetlands NWI: Shapefile'
-  updates:
-    - March 2017
+    - Wetlands will be updated approximately twice per year when NWI releases new updates. The current version is up-to-date as of October 15, 2018.
 ---
 <figure class="caption caption--right">
   <img class="caption__image" title="map" src="{{ "/images/wetlands.png" | prepend: site.baseurl }}" alt="Wetlands" />
   <figcaption class="caption__text">Wetlands in the Bear River Migratory Bird Refuge Area</figcaption>
 </figure>
-{% include abstract.html download="https://drive.google.com/drive/folders/0ByStJjVZ7c7mOXdndHpnektzV0U"
+{% include abstract.html download="https://drive.google.com/drive/u/0/folders/0ByStJjVZ7c7mNW9iXzJrQUYyV0U"
 name="Wetlands"
 stewards="Utah Geologic Survey, Department of Natural Resources"
-abstract="Water.Wetlands was compiled from the US Fish and Wildlife Service's National Wetland Inventory (NWI) with additional data from US Forest Service, Utah Geological Survey, and Utah Automated Geographic Reference Center using the Cowardin (1979) classification scheme. In addition to the Cowardin fields, data has been attributed by the Utah Geological Survey to provide functional and descriptive codes associated with each wetland class as described by the <a href='http://ugspub.nr.utah.gov/publications/open_file_reports/ofr-661.pdf'>Utah Wetland Functional Classification: Version 1</a>. This data set is maintained by the Utah Geologic Survey and will be updated as new mapping projects are completed." %}
+abstract="Wetland location and classification polygon map data, derived from the U.S. Fish and Wildlife Service’s <a href="https://www.fws.gov/wetlands/">National Wetlands Inventory</a> (NWI), including information on wetlands, shallow and deep water, and riverine features. This layer will be updated approximately twice per year following the update schedule set by NWI." %}
```

Comment:
```suggestion
abstract="Wetland location and classification polygon map data, derived from the U.S. Fish and Wildlife Service’s <a href='https://www.fws.gov/wetlands/'>National Wetlands Inventory</a> (NWI), including information on wetlands, shallow and deep water, and riverine features. This layer will be updated approximately twice per year following the update schedule set by NWI." %}
```

double quotes changed to single because this text is already inside of doubles...